apps/bot_manager/lib/bot_state_machine_checker.ex

```diff
@@ -0,0 +1,67 @@
+defmodule BotManager.BotStateMachineChecker do
+  @moduledoc """
+  This module will take care of deciding what the bot will do on each deciding step
+  """
+  defstruct [
+    # The bot state, these are the possible states: [:idling, :moving, :aggresive, :running_away]
```

Comment:
Not important. But maybe we can change :aggresive to :attacking to match the other verbs.

Reply:
Sure!